base_wamas_ubl/lib/wamas/utils.py

```diff
@@ -0,0 +1,40 @@
+import codecs
+import os
+from datetime import datetime
+
+from dateutil.parser import parse
+
+
+def file_path(path):
+    return os.path.join(os.path.abspath(os.path.dirname(__file__)), path)
+
+
+def file_open(path):
+    return codecs.open(path, "r", "iso-8859-1")
```

Comment:
- although not explicitly stated in the wamas specs, all real life wamas files encountered so far were encoded in iso-8859-1
- we choose not to use `chardet` here because if a file happened to be incorrectly detected with a different encoding one day, it would impact its parsing as the wamas grammar uses fixed-width fields, and this could lead to incorrect data silently introduced in the system